server/middlewares/similar-packages/fixtures.js

```diff
@@ -232,7 +232,7 @@ const categories = {
       { tag: 'escape', weight: Weight.MID },
       { tag: 'filter', weight: Weight.NORMAL },
       { tag: 'xss', weight: Weight.HIGH },
-      { tag: 'whitelist', weight: Weight.SMALL },
+      { tag: 'allowlist', weight: Weight.SMALL },
```

Comment:
Perhaps this should remain `tag: 'whitelist'` if it refers to https://www.npmjs.com/search?q=whitelist